docs/design/software-design-specification.adoc

```diff
@@ -217,7 +217,7 @@ Input parameters: TODO
 
 | SDS-TASK-14
 | `ods-python-toolset` contaner image
-| Container image to build Python applications. Based on `ubi8/ubi-minimal` (SDS-EXT-2), includes SDS-EXT-14, SDS-EXT-15, SDS-SHARED-3, SDS-TASK-15 and SDS-TASK-27.
+| Container image to build Python applications. Based on `ubi8/python-39` (SDS-EXT-28), includes SDS-EXT-14, SDS-EXT-15, SDS-SHARED-3, SDS-TASK-15 and SDS-TASK-27.
```

Comment:
Now we can cut `SDS-EXT-14, SDS-EXT-15` from this list, and `SDS-EXT-14` also from the table below I believe.

Reply:
I believe `SDS-EXT-15` can be also removed from the table below since is no longer referenced anywhere, or we want to keep a general reference to the `tar` tool?
Also, if I remove for example `SDS-EXT-14` from the table below, shall I renumber the list? or we keep `14` gone? I would go for renumbering...

Reply:
also, if we are removing `SDS-EXT-14` shouldn't we remove similar ones like `SDS-EXT-{10-12,16}` ?

Reply:
* `tar` is still used in `Dockerfile.helm` so we should keep `SDS-EXT-15` in the list
* We should not change numbers. There just will be no `SDS-EXT-14` going forward.
* `SDS-EXT-10` is indeed unused and should be removed. `SDS-EXT-12` is still in use and correct I believe.
* `SDS-EXT-16` is also unused but still referenced - both should be fixed

Thanks!